### PR TITLE
feat: community page divided in subpages

### LINKED ISSUE #116: Create a "Community" section
Make a community section with links to articles and talks about pnpm.

Similar to https://reactjs.org/community/support.html or https://docusaurus.io/community/support. The links should be take from https://github.com/pnpm/awesome-pnpm

Blocked until https://github.com/facebook/docusaurus/issues/3810 is fixed in docusaurus.

### PR DESCRIPTION
**Pull request by olrtg**
Fixes #116

### CODE REVIEW

**Review by hunter2009 — APPROVED** [review]

File: sidebarsCommunity.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "community": [
+    "articles",
+    "podcasts",
+    "videos",
+    "packages",
+    "toolings"
+  ]
```
Comment (zkochan) [inline sidebarsCommunity.json:8]: let's merge these to pages into one. Just "tools"

File: community/toolings.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+id: toolings
+title: Toolings with pnpm support
```
Comment (zkochan) [inline community/toolings.md:3]: Let's just name it "Tools"

**Review by zkochan — COMMENTED** [review]
There are conflicts

**Comment by olrtg** [comment]
If there's something else to change please let me know!
Also how can we resolve this conflict? Is this something that i should do on my end? Or you'll take care of that?

**Pull request merged**